docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+
+[3] - Automatic archive import is recommended when restoring a backup set taken from a different environment.
+
+[4] - Archive seed restore is not deterministic. Once the restore request is created, the first SM that starts or is restarted will perform the restore of its archive.
+
+For detailed information about features, enhancements, and fixed problems, please check [NuoDB Release Notes](https://doc.nuodb.com/nuodb/latest/release-notes/) and [NuoDB Helm Charts Release Notes](https://github.com/nuodb/nuodb-helm-charts/releases)
+
+## Backup and Restore Mechanisms
+
+### Scheduled online database backups
+
+Multiple backup cron jobs are scheduled as part of the [database](../stable/database) chart deployment by default.
+The backup schedules can be customized to meetdifferent backup and recovery objectives.
+They are exposed as Helm chart values in particular:
+- `database.hotCopy.fullSchedule` - configures the schedule of _full_ backup Kubernetes job
+- `database.hotCopy.incrementalSchedule` - configures the schedule of _incremental_ backup Kubernetes job
+- `database.hotcopy.journalBackup.intervalMinutes` - configures the schedule of _journal_ backup Kubernetes job. _journal_ hot copy must be explicitly enabled using `database.hotcopy.journalBackup.enabled`.
+
+[Cron expression format](https://en.wikipedia.org/wiki/Cron) documents the format of the first two schedules listed above.
+
+An HC SM has a backup volume attached and is selected for hot copy operation during a scheduled backup.
+Each _full_ hot copy will create a backup set with the same name in the backup root directory (by default `/var/opt/nuodb/backup`) on all HC SMs in a single database and a single backup group.
+By default, all HC SMs in a single Kubernetes cluster are part of one backup group.
+This can be adjusted by changing the `database.hotcopy.backupGroup` value for multiple Kubernetes clusters in multi-cluster deployment.
+_Incremental_ and _journal_ hot copies are always stored in the current backup set if one exists.
+If there is no current backup set for some of the archives served by a HC SM when _incremental_ or _journal_ backup is requested, a _full_ hot copy is triggered automatically.
+The `nuobackup` script is made available as a configMap and is used by all backup cron jobs to send a hot copy request of the specified type to the HC SMs.
+The _Full_ hot copy result is recorded into the NuoDB Admin domain key-value (KV) store upon successful finish.
+
+The backup jobs execution should be monitored on a regular basis to ensure that they complete successfully.
+By default, each backup job execution will be retried on failure which can be configured by the  `database.hotCopy.restartPolicy` setting.
+There are no backup retention policies currently available. The admin user is responsible to make sure that there is enough space in the backup volume of each SM for new hot copies to be created.
```

Comment:
This is really important info an feels a little lost between the other two sentences about the job histories. Can we pull it into a section on retention management?  Where we could also mentioned the possibility of using customer provided read write many volumes (EFS, azure file) enabling retention management more easily using cloud tooling for example.

Reply:
Good call! I don't know a lot about the "cloud tooling" mentioned here.
I will include the statement as is in a new section. If you would like me to include more info, please let me know.